code/modules/SCP/SCPs/SCP-2427-3.dm

```diff
@@ -206,6 +206,7 @@ GLOBAL_LIST_EMPTY(scp2427_3s)
 	return ..()
 
 /mob/living/simple_animal/hostile/scp_2427_3/SelfMove(direction)
+	resting = FALSE //2427 is forced to rest on dying... So make them not rest the next time they move.
```

Comment:
leave this one though, this actually is a good comment to put

Reply:
(you dont need to do anything for this just hit resolve on this one)

Reply:
this looks stupid

Reply:
yeah it does dosent it